# *Convolutions*

# Definition 7.2

### Let $X$ and $Y$ be two continuous random variables with density functions $f(x)$ and $g(y)$

### Then, the *convolution* $f*g$ of $f$ and $g$ is defined as:

# $(f*g)(\alpha_{0}) = \int_{-\infty}^{\infty}f(\alpha_{0}-y)g(y)dy = \int_{-\infty}^{\infty}f(x)g(\alpha_{0}-x)dx$

____

# Theorem 7.1

### Let $X$ and $Y$ be two independent random variables with density functions $f_{X}(x)$ and $f_{Y}(y)$

### Then the sum $Z=X+Y$ is a random variable with density function $f_{Z}(z)$, where $f_{Z}$ is the convolution of $f_{X}$ and $f_{Y}$

___

# *Sum of Two Independent Uniform Random Variables*

## Example

### Let $X$ and $Y$ be random variables distributed uniformly on $[0,1]$ and $Z=X+Y$

### From the definition above, we know that 

# $f_{Z}(z) = \int_{-\infty}^{\infty}f_{X}(z-y)f_{Y}(y)dy$

### We know $f_{X}(x) = f_{Y}(x) = 1$ when $0\leq x \leq 1$

### The minimum value for $Z$ is 0, and the maximum is 2

### Let's break $0\leq z \leq 2$ into two parts: i) $[0,1]$ and $[1,2]$

### i)

### For $f_{X}(z-y)$ and $f_{Y}(y)$ to be non-zero, we need $0 \leq z-y$ and $0\leq y \implies 0 \leq y \leq z$

### Therefore on $[0,1]$, $f_{Z}(z) = \int_{0}^{z}dy = z$

### ii)

### For $f_{X}(z-y)$ and $f_{Y}(y)$ to be non-zero, we need $z-y \leq 1 \implies z-1 \leq y$ and $y<1 \implies z-1 \leq y \leq 1$

### Therefore on $[1,2]$, $f_{Z}(z) = \int_{z-1}^{1}dy = 1 - (z-1) = 2-z$

### So the expression for $f_{Z}(z)$ is given by:

# $f_{Z}(z) = \left\{\begin{matrix} z & \text{ ; if }z\leq 1\\ 2-z & \text{ ; if }z>1 \end{matrix}\right.$

_____

# *Sum of Two Independent Exponential Random Variables*

## Example

### We choose two values $X$ and $Y$ at random on $[0, \infty)$ with an exponential density and parameter $\lambda$

### What is the density of $Z = X+Y$?

### We know $f_{X}(x) = f_{Y}(x) = \lambda e^{-\lambda x}$ when $x\geq 0$

### Using the definition above, we get $f_{Z}(z) = \int_{0}^{z}\lambda e^{-\lambda(z-y)} \cdot \lambda e^{-\lambda y}dy = \lambda^{2}\int_{0}^{z}e^{-\lambda (z-y+y)}dy = \lambda^{2}e^{-\lambda z}z$

____

# *Sum of Two Independent Normal Random Variables*

## Example

### It is an interesting and important fact that the convolution of two normal densities with means $\mu_{1}$ and $\mu_{2}$ and variances $\sigma^{2}_{1}$ and $\sigma^{2}_{2}$ is *also a normal distribution* with $\mu = \mu_{1}+\mu_{2}$ and $\sigma^{2} = \sigma^{2}_{1}+\sigma^{2}_{2}$

### We'll show this for two standard normal distributions: If $X$ and $Y$ are independent random variables with standard normal density, then

# $f_{X}(x) = f_{Y}(y) = \frac{1}{\sqrt{2\pi}}e^{-\frac{x^{2}}{2}}$

### So if $Z=X+Y$ then

# $f_{Z}(z) = \int_{-\infty}^{\infty}\frac{1}{2\pi}e^{-\frac{(z-y)^{2}}{2}}e^{-\frac{y^{2}}{2}}dy = ... = \frac{1}{\sqrt{4\pi}}e^{\frac{-z^{2}}{4}}$

_______

# *Chi-Squared Density*

## The sum of the squares of $n$ independent standard normally distributed random variables has a gamma density with $\lambda = \frac{1}{2}$ and $\beta = \frac{n}{2}$

## This special density is called the *Chi-Squared density*

### An important use for the chi-squared density is to test experimental data against theoretical values

#### This tells us whether the data supports the theoretical model

## Let $X$ be the random variable which represents a theoretical outcome in the model of the experiment, and let $m(x)$ be the distribution function of $X$

## We then calculate the expression $V = \sum_{x}\frac{(o_{x} - n\cdot m(x))^{2}}{n\cdot m(x)}$ where each $x$ is a possible outcome, $n$ is the total number of possible outcomes, and $O_{x}$ is the number of outcomes of type $x$ observed in the data

## For moderate to large values of $n$, we say that $V$ is approximately chi-squared distributed

___

## Example

### Suppose we're given a single die

### We want to test whether the die is fair

### So our theoretical density is the discrete uniform density on [1,6]. If we roll the die $n$ times, we'd expect to get around $\frac{n}{6}$ of each side

### Then, $o_{1}$ denotes the number of observed rolls that got a 1, $o_{2}$ the number of observed rolls for 2, ...

# $V = \sum_{x}\frac{(o_{x} - n\cdot \frac{1}{6})^{2}}{n\cdot \frac{1}{6}} = \frac{(o_{1} - \frac{n}{6})^{2}}{\frac{n}{6}} + ... + \frac{(o_{6} - \frac{n}{6})^{2}}{\frac{n}{6}}$

### Since there are 6 possible faces, if we know $o_{1}, o_{2}, ..., o_{5}$ then we automatically know $o_{6}$ since it's just $n - (o_{1}+o_{2}+ ...+o_{5})$

- In other words, the only values that can vary are $o_{1}, o_{2}, ..., o_{5}$

### Therefore there are *5 degrees of freedom*

### Now, suppose we roll the die 60 times ($\implies n = 60$) with outcomes described below:

![](images/table7-1.PNG)

### Then, we calculate the value of $V$:

In [13]:
val = 60/6
V = 0

for oi in [15,8,7,5,7,18]:
    Vi = ((oi-val)**2)/float(val)
    V += Vi

In [16]:
V

13.600000000000001

### We look at the plot of the chi-square distribution with 5 degrees of freedom to see how likely/unlikely it is that we would get a $V$ value of 13.6 for a *fair coin*

![](images/chi2-distn.PNG)

### As we can see, the vast majority of the distribution is to the left of 13.6 (i.e. the probability of that outcome for a fair die is pretty unlikely)

_______

# *Independent Trials*

### Consider $n$ independent random variables all with the same density function

### If $S_{n} = X_{1}+X_{2}+...X_{n}$ then $f_{S_{n}} = (f_{1}*f_{2}*...*f_{n})(x)$

- **Note**: this is a huge pain in the ass to calculate for large $n$

___

## Example

### Suppose $X_{1}, X_{2},...,X_{n}$ are uniformly distributed on $[0,1]$

### Then 

# $f_{S_{n}} = \frac{1}{(n-1)!}\sum_{0\leq j \leq x}(-1)^{j}\binom{n}{j}(x-j)^{n-1}$ for $0<x<n$